### PR TITLE
Fix enabling realtime in table editor not working if previously realtime was disabled entirely from database/replications

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

**Comment by joshenlim** [comment]
Hold this sorry - got the fix wrong

**Comment by joshenlim** [comment]
okay good to go now

File: studio/stores/pgmeta/MetaStore.ts [hunk 1]
```diff
@@ -265,6 +265,46 @@ export default class MetaStore implements IMetaStore {
     return await this.query(query)
   }
 
+  async updateTableRealtime(table: PostgresTable, enable: boolean) {
+    const publications = this.publications.list()
+    const publicTables = this.tables.list((table: PostgresTable) => table.schema === 'public')
+
+    const realtimePublication = publications.find((pub) => pub.name === 'supabase_realtime')
+    const { id, tables: publicationTables } = realtimePublication
```
Comment (alaister) [inline studio/stores/pgmeta/MetaStore.ts:273]: Is this destructuring safe to do here? What if it doesn't find the `supabase_realtime` publication?

**Review by alaister — COMMENTED** [review]
Looks good! Seems much cleaner too 👌
Just one question before this goes in.

File: studio/stores/pgmeta/MetaStore.ts (hunk 1 above)
Comment (joshenlim) [reply studio/stores/pgmeta/MetaStore.ts:273]: oh yeah you're right actually - lemme push a small update to make it a bit more fault resistant

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**